lib/gemfury/command/app.rb

```diff
@@ -40,9 +42,16 @@ def versions(gem_name)
     with_checks_and_rescues do
       versions = client.versions(gem_name)
       shell.say "\n*** #{gem_name.capitalize} Versions ***\n\n"
+
+      va = []
+      va << [ 'version', 'authors', 'uploaded' ]
       versions.each do |v|
-        shell.say v['version']
+        authors = v['authors'].join(', ')
+        uploaded = Time.parse(v['updated_at']).strftime('%F %R %z')
```

Comment:
`'authors'` is just package metadata. To match the dashboard, you'll need to switch that to 'created_by' (need to expose this via API first). Also, we use `created_at  for "Uploaded at" on the dashboard, not `updated_at`.